basics/email_validation.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+require 'creds.php';
+require 'requests.php';
+
+/**
+ * Checks if the email is valid or not.
+ *
+ * @param string $email_address
+ *   User's email address.
+ *
+ * @return bool
+ *   Returns true if email address is valid else returns false.
+ */
+function isEmailValid(string $email_address): bool {
+  global $api_key;
+  $url = 'http://apilayer.net/api/check?access_key=' . $api_key . '&email=' . $email_address;
+
+  // Calling request method to call the api, get json file and parse it.Then parsing the json and storing values as associative array.
```

Comment:
Break this into 2 lines.
Normally a line should not have more than 80 characters in comment.